feat: Add GitHub pullrequest template (#141)
aep-dev/aep.dev: .github/PULL_REQUEST_TEMPLATE.md

## 📑 Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## 🍱 Types of changes

What types of changes does your code introduce to AEP?
_Put an `x` in the boxes that apply_

> - [ ] Enhancement

Reviewer: Should we also have a checkbox for adopting an AIP from google.aip.dev?  It might be interesting to distinguish that from an entirely new proposal.

Author: I haven't thought of this one. I fully agree it makes sense during the initial phase.

- [ ] [New proposal](https://aep.dev/1#workflow)

## 📋 Your checklist for this pull request
Please review the [AEP Style and Guidance](https://aep.dev/style-guide) for contributing to this repository.

### General

> - [ ] Ensure that basic [Guidance](https://aep.dev/style-guide#guidance) is met.

Reviewer 2: ```suggestion
- [ ] Basic [Guidance](https://aep.dev/style-guide#guidance) is met.
```

nit - just for your consideration. But the "ensure" word doesn't really add a lot of value here - it's a checklist, so by checking it you're implicitly saying you've done what the line has asked you to do.

If you do change it I'd suggest changing it for all lines.

> - [ ] Ensure to get the [Referencing AEPs](https://aep.dev/style-guide#referencing-aeps) right.

Reviewer: ```suggestion
- [ ] Ensure that your PR [references AEPs](https://aep.dev/style-guide#referencing-aeps) correctly.
```

Nit: grammar/phrasing

> ### Additional for new proposal

Reviewer 2: ```suggestion
### Additional checklist for a new AEP
```

I think we've agreed upon "AEP" being the nouns - although proposal is not incorrect.

> - [ ] A new AEP should be no more than two pages if printed out.

Reviewer: ```suggestion
- [ ] A new AEP **should** be no more than two pages if printed out.
```

The site generator will color the word "should" orange if formatted like this.
